### DISCUSSION TITLE
Add a CONNECT button on the models.

### DISCUSSION THREAD

**Discussion by MancV21** [LykosAI/StabilityMatrix#71] [Ideas]
Hi, just started using the tool, loving it so far but because of the fragility of CIVITAI, which can go down for a few seconds to hours at any time, when importing some models, your program is unable to get the info on some models if you're importing at the 'wrong' time. So an option to just copy the models across into the SM model folders, and then be able to right-click a model and have a CONNECT (or something similar) option that would then pull the info for that model from CIVITAI at a later date when it's working OK.

**Comment by ionite34** [comment]
Will add soon 👍

**Comment by mohnjiles** [comment]
Hi, we added a feature that lets you "Find Connected Metadata" for each of the folders in the Checkpoints tab. It will re-scan the entire folder  & download the appropriate metadata. You can find the option by right-clicking one of the folders in the Checkpoints tab. 

It should be available in [v2.4.0 or later](https://github.com/LykosAI/StabilityMatrix/releases/tag/v2.4.6)

Thanks for the suggestion!

**Reply by Kryptonic83** [reply]
ah, thanks, was looking for that by right clicking on individual model instead of clicking on the folder/section instead.
